src/mbgl/map/tile_data.cpp

```diff
@@ -34,15 +36,20 @@ void TileData::setState(const State& state_) {
     state = state_;
 }
 
-void TileData::request(Worker& worker, float pixelRatio, std::function<void()> callback) {
+void TileData::request(Worker& worker,
+                       float pixelRatio,
+                       const std::function<void()>& successCallback,
+                       const std::function<void(const std::string& message)>& failureCallback) {
```

Comment:
> Should we use the single callback convention here too?

Yes, I will refactor this one in another branch. I've to find a nice way of replacing this one because the Worker expects a std::function<void()> as callback and I want to report errors during the tile parsing.